style.css

```diff
@@ -0,0 +1,122 @@
+/* Remove all padding and margin from the browser */
+* {
+  padding: 0;
+  margin: 0;
+}
+
+/* Lays out all page elements in a flex container */
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: stretch;
+  height: 100vh;
+}
+
+/* Sets the initial height of the map container */
+.map-container {
+  height: 90vh;
+  flex-shrink: 3;
+  min-height: 0 ;
+}
+
+/* The map will fill it's container */
+#map {
+  height: 100%;
+}
+
+header {
+  background-color: gray;
+  color: whitesmoke;
+}
+
+h1 {
+  font-family: helvetica;
+  padding: 5px;
+}
+
+/* Styles the aside which is the expandable div. Position relative allows for the 
+double up arrow to stay in the same place once expanded */
+aside {
+  background-color: orange;
+  flex-grow: 3;
+  position: relative; 
+  padding: 10px;
+}
+
+/* Allows for the restaurant links to stack vertically as blocks */
+a {
+  display: block;
+  text-decoration: none;
+  color: whitesmoke;
+  font-family: helvetica;
```

Comment:
since your `font-family` is always helvetica you could just set it on the body and save yourself a line or two of code.